red-hat-gallery/openshift-pipelines-gallery/base/adservice-pipeline.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     default: adservice
   - name: GIT_REPO
     type: string
-    default: https://github.com/noseka1/red-hat-gallery
+    default: https://github.com/stratus-ss/red-hat-gallery
```

Comment:
Let's not change the noseka1 to stratus-ss. After we find a GitHub organization for hosting red-hat-gallery, we can update it here. Until that time let's leave it to noseka1.